app/client/components/messageBar.tsx

```diff
@@ -18,7 +18,7 @@ export default function MessageBar() {
         transitionProperty: "all",
         transitionDuration: "1.5s",
         transitionTimingFunction: "cubic-bezier(0, 1, 0.5, 1)",
-        top: message.showMessage ? 0 : -35,
+        top: message.showMessage ? "auto" : -35,
```

Comment:
Ensures message bar is visible when google translate extension is active
<img width="1792" alt="Screen Shot 2020-09-25 at 5 35 16 pm" src="https://user-images.githubusercontent.com/27039316/94239868-c889d280-ff55-11ea-9692-0085275f658d.png">